scripts/codebuild/build.sh

```diff
@@ -13,4 +13,4 @@ echo "BUILD SEARCH URL: ${SEARCH_URL}"
 echo "BUILD SEARCH INDEX: ${SEARCH_INDEX}"
 
 # build
-yarn workspace site build || { echo "Gatsby build failed" ;exit 1; }
+yarn workspace site build &> test_results.log || { echo "\e[31mGatsby build failed${reset}"; cat build.log; exit 1; } 
```

Comment:
Can we also echo to the console the location of the log?

Reply:
well.  that would not work that well on the live server as it will be destroyed when the build is done.  I really need to send the files to s3 somewhere.  Locally they will be in the root of the project if you run build scripts but it is supposed to print the error to standard out when the script is unsuccessful.